docs/api/auto-updater.md

```diff
@@ -32,10 +32,14 @@ This is a requirement of `Squirrel.Mac`.
 ### Windows
 
 On Windows, you have to install your app into a user's machine before you can
-use the `autoUpdater`, so it is recommended that you use the
-[electron-winstaller][installer-lib], [Electron Forge][electron-forge-lib] or the [grunt-electron-installer][installer] package to generate a Windows installer.
+use the `autoUpdater`, so it is recommended that you use
+[electron-winstaller][installer-lib] or [Electron Forge][electron-forge-lib] to generate a Windows installer.
 
-When using [electron-winstaller][installer-lib] or [Electron Forge][electron-forge-lib] make sure you do not try to update your app [the first time it runs](https://github.com/electron/windows-installer#handling-squirrel-events) (Also see [this issue for more info](https://github.com/electron/electron/issues/7155)). It's also recommended to use [electron-squirrel-startup](https://github.com/mongodb-js/electron-squirrel-startup) to get desktop shortcuts for your app.
+When using [electron-winstaller][installer-lib] or [Electron Forge][electron-forge-lib] make sure you do not try to update your app
+[the first time it runs](https://github.com/electron/windows-installer#handling-squirrel-events)
+(Also see [this issue for more info](https://github.com/electron/electron/issues/7155)).
+It's also recommended to use [electron-squirrel-startup](https://github.com/mongodb-js/electron-squirrel-startup)
+to get desktop shortcuts for your app.
```

Comment:
I think this is unclear in it's phrasing, it focuses heavily on "do not update on first run" which isn't correct without the context of squirrel events.

We should probably instead phrase "ensure you are handling squirrel events correctly, we reccomend using electron-squirrel-startup which does this for you, so you don't try to update your app during a squirrel launch event"

Reply:
This is some legacy phrasing that I had to touch because I wanted to remove references to the grunt tooling that was mentioned but happy to amend this as well!

Reply:
I just took a look at how this is implemented in various places and `electron-squirrel-startup` actually doesn't handle `--squirrel-firstrun` at all.

The ugly way we work around this in Fiddle is to actually delay the update check by 10s on startup:

https://github.com/electron/fiddle/blob/f89d8d2b45b1f8ccb2bfa294c48088dadc292abe/src/main/update.ts#L5-L14

Reply:
[90b0b55](https://github.com/electron/electron/pull/43234/commits/90b0b55f82660b3295c70e286ab0768c8cdd0ed9)

Attempted to fix the wording here but could use another review